src/main/java/com/spotify/docker/client/messages/ContainerConfig.java

```diff
@@ -372,12 +372,20 @@ public Builder volumes(final String... volumes) {
     @JsonProperty("Retries")
     public abstract Integer retries();
 
+    /**
+     * In nanoseconds.
+     */
+    @Nullable
+    @JsonProperty("StartPeriod")
+    public abstract Long startPeriod();
+
     @JsonCreator
     public static Healthcheck create(
             @JsonProperty("Test") final List<String> test,
             @JsonProperty("Interval") final Long interval,
             @JsonProperty("Timeout") final Long timeout,
-            @JsonProperty("Retries") final Integer retries) {
+            @JsonProperty("Retries") final Integer retries,
+            @JsonProperty("StartPeriod") final Long startPeriod) {
```

Comment:
I like what @johnflavin said; we can have multiple `create()` factory methods, and adding overloads is a nice way to deal with the addition of new (optional) parameters. There can only be one `@JsonCreator` annotated method but that doesn't mean you can't have other `public static` factories.